blazor/predefined-dialogs/position.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Positioning in Blazor Predefined Dialog Component | Syncfusion
+description: Checkout and learn here all about Positioning in Syncfusion Predefined Blazor Dialog component and much more details.
+platform: Blazor
+control: Predefined Dialogs
+documentation: ug
+---
+
+# Positioning in Blazor Predefined Dialogs Component
+
+You can customize the dialog position by using the [DialogOptions.Position](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogOptions.html#Syncfusion_Blazor_Popups_DialogOptions_Position)  property. 
```

Comment:
Need to give default position value, and also give what are the values this API accepts?

Reply:
Added the default values